src/components/CarouselCard/index.js

```diff
@@ -5,7 +5,7 @@ import classnames from 'classnames';
 import { Provider } from './context';
 import Indicators from './indicators';
 import AnimationButton from './animationButton';
-import { getItemIndex, getChildTabNodes, insertChildOrderly } from './utils';
+import { getItemIndex, getChildTabNodes, insertChildOrderly, carouselCardContainerStyles } from './utils';
```

Comment:
getContainerStyles since it is a function, would be carouselCardContainerStyles if it was a const.